### PR TITLE
Add notes how to use bot accounts with slash commands

### PR DESCRIPTION
**Pull request by hanzei**
Follow up from https://community.mattermost.com/core/pl/5rpah9eis3yp3gf4qtcemgyswe

### CODE REVIEW

**Comment by hanzei** [comment]
http://mattermost-developer-documentation.s3-website-us-east-1.amazonaws.com/commits/0bc84c93e13e43c7d12d4a1f86c0e7a401abadd8/integrate/slash-commands/#how-do-i-use-bot-accounts-to-reply-to-slash-commands for a quick review.

File: site/content/integrate/slash-commands/_index.md [hunk 1]
```diff
@@ -183,3 +183,13 @@ Ensure you are emitting the `Content-Type: application/json` header, otherwise y
 #### Are slash commands Slack-compatible?
 
 See the [admin guide's notes on Slack compatibillity](https://docs.mattermost.com/developer/slash-commands.html#slack-compatibility).
+
+#### How do I use Bot Accounts to reply to slash commands?
+
+##### If you are developing a integration
```
Comment (jasonblais) [inline site/content/integrate/slash-commands/_index.md]:
```suggestion
##### If you are developing an integration
```

File: site/content/integrate/slash-commands/_index.md [hunk 2]
```diff
@@ -183,3 +183,13 @@ Ensure you are emitting the `Content-Type: application/json` header, otherwise y
...
+- Set up a [Personal Access Token](https://docs.mattermost.com/developer/personal-access-tokens.html) for the Bot Account you want to reply with.
```
Comment (jasonblais) [inline site/content/integrate/slash-commands/_index.md]: This section might change with 5.12 release (as there will be a different flow for creating personal access token for bot), but this is good for now.

File: site/content/integrate/slash-commands/_index.md [hunk 3]
```diff
@@ -183,3 +183,13 @@ Ensure you are emitting the `Content-Type: application/json` header, otherwise y
...
+- Use the [REST API](https://api.mattermost.com/) to create a post with the Access Token.
```
Comment (jasonblais) [inline site/content/integrate/slash-commands/_index.md]: Is there a specific section we can link to, e.g. https://api.mattermost.com/#tag/posts%2Fpaths%2F~1posts%2Fpost

Comment (jasonblais) [inline site/content/integrate/slash-commands/_index.md]:
```suggestion
- Use the [REST API](https://api.mattermost.com/#tag/posts%2Fpaths%2F~1posts%2Fpost) to create a post with the Access Token.
```

File: site/content/integrate/slash-commands/_index.md [hunk 4]
```diff
@@ -183,3 +183,13 @@ Ensure you are emitting the `Content-Type: application/json` header, otherwise y
...
+
+##### If you are developing a plugin
+
+Use [`CreatePost`](https://developers.mattermost.com/extend/plugins/server/reference/#API.CreatePost). Make sure to set the  `UserId` of the post to the `UserId` of the Bot Account. If you want to want to create an ephemeral post, use [`SendEphemeralPost`](https://developers.mattermost.com/extend/plugins/server/reference/#API.SendEphemeralPost) instead.
```
Comment (jasonblais) [inline site/content/integrate/slash-commands/_index.md]:
```suggestion
Use [`CreatePost`](https://developers.mattermost.com/extend/plugins/server/reference/#API.CreatePost) plugin API. Make sure to set the  `UserId` of the post to the `UserId` of the Bot Account. If you want to create an ephemeral post, use [`SendEphemeralPost`](https://developers.mattermost.com/extend/plugins/server/reference/#API.SendEphemeralPost) plugin API instead.
```

**Pull request merged**